lib/PuppeteerSharp.Tests/Issues/PdfIssuesTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Issues
+{
+    public class PdfIssuesTests : PuppeteerBaseTest
+    {
+        public PdfIssuesTests()
+        {
+            Downloader.CreateDefault().DownloadRevisionAsync(TestConstants.ChromiumRevision).GetAwaiter().GetResult();
+        }
+
+        [Fact]
+        public async Task Issue100()
```

Comment:
What do you think about this @Meir017?
Should we add every issue we find? Should those be enabled by default?

Reply:
maybe each file should cover a single issue - like this https://github.com/JamesNK/Newtonsoft.Json/tree/master/Src/Newtonsoft.Json.Tests/Issues ?